drivers/led/lp5562.c

```diff
@@ -310,7 +310,7 @@ static int lp5562_set_led_source(const struct device *dev,
 				 enum lp5562_led_channels channel,
 				 enum lp5562_led_sources source)
 {
-	const struct lp5562_config *config = dev->config;
+	const struct lp5562_config *const config = dev->config;
```

Comment:
note: adding `const` here is not a change related to adding support for multiple instances. if you prefer marking this pointer as const, please add the changes in a separate commit.

Reply:
Will do, thanks.